app/extensions/ipfs/js/main.js

```diff
@@ -0,0 +1,11 @@
+// const ipc = window.chrome.ipcRenderer
+
+chrome.protocol.registerStringProtocol('ipfs', (request, callback) => {
+  /*
+  const node = new IPFS()
+  node.on('ready', () => {
+    callback('hi there!' + test() + IPFS) // eslint-disable-line
+  })
+  */
+  callback('hi there!' + test() + IPFS) // eslint-disable-line
```

Comment:
I'm being unable to create an IPFS node as the background process crashes (for some reason) and I have no way to inspect it. Looking for a better way to debug this background extension.